### PR TITLE
Add arm64 lts-4.9.75 and nxp layerscape platforms

### PR DESCRIPTION
**Pull request by shengzhou**
execute following build to generate new opennetworklinux/builder9:1.1
cd docker/images/builder9/1.1
make
make push

then, run following build to generate builds/arm64/installer/swi/builds/ONL-2.0.0_ONL-OS_<timestamp>_ARM64_SWI_INSTALLER
export VERSION=9; make docker

Verified on NXP arm64 platforms.

### CODE REVIEW

**Comment by shengzhou** [comment]
@jnealtowns  Hi jnealtowns, do you have any comment for this pull request?  with new builder9:1.1, it worked fine for building both arm64 and amd64 with verification.  could you merge this pull request?

**Comment by jnealtowns** [comment]
@shengzhou Sorry about the delay on this one. We will review shortly.

File: packages/platforms/nxp/arm64/arm64-nxp-layerscape/platform-config/r0/PKG.yml [hunk 1]
```diff
@@ -0,0 +1,80 @@
+############################################################
+#
+# PKG template for all platform-config packages.
+#
+############################################################
+variables:
+  LS2088: arm64-nxp-ls2088ardb-r0
```
Comment (jnealtowns) [inline packages/platforms/nxp/arm64/arm64-nxp-layerscape/platform-config/r0/PKG.yml]: @shengzhou All of these should be declared as separate platforms under arm64

**Comment by shengzhou** [comment]
@jnealtowns  This pull request has been updated with separate platforms under arm64, please review and merge it.

**Pull request merged**